test/README.md

```diff
@@ -1,130 +1,153 @@
-Test OpenSSL
-============
+How to add recipes
+==================
```

Comment:
Ewwww, I'm really not very keen on mixing stuff geared for OpenSSL developers with stuff for OpenSSL users.  If I was a user, having to go through all the stuff for developers would feel quite jarring.

Reply:
This was a merge glitch that I was about to correct.
Are you okay if I move this section to the end of `test/README.md`,
or which again was the file name you were proposing (can't find it at the moment)?

Reply:
I would prefer a separate file name for the developer stuff.  Sorry, I don't remember what file name I proposed

Reply:
Fine, so I've moved to info on adding tests from `test/README.md` to a new file `test/README-dev.md` 
and referenced it  in `INSTALL.md`.